app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+  def markdown(text)
```

Comment:
@no-reply can you tell me more about your thoughts on this? I'm not sure I understand what you are suggesting.